src/App.tsx

```diff
@@ -105,19 +120,20 @@ const App: React.FC = () => {
       );
       const treeJSON = await treeRes.json();
 
-      const allItems = treeJSON["tree"].length;
+      const numberOfItems = treeJSON["tree"].length;
```

Comment:
TREE is a magic string, what does tree stand for?
const GITHUB_TREE?

Reply:
@ralph-dev so resJSON objects has these keys ["commit"]["tree"]["sha"]...
isn't it too much to name these keys on top of the file?